src/encoded/tests/test_search.py

```diff
@@ -5,54 +5,13 @@
 
 
 def test_search_view(workbook, testapp):
```

Comment:
let's add a test to ensure /search automatically as ?type=Item if nothing specified